src/kuzya.cpp

```diff
@@ -52,7 +47,7 @@
 #include "helpbrowser.h"
 #include "translator.h"
 #include "version.h"
-
+#include "kuzyaabout.hpp"
```

Comment:
I'll also suggest to find better name for class and files (aboutdialog.*, aboutkuzya.*) -it sounds a bit more natural.